api/include/mfxstructures.h

```diff
@@ -2269,6 +2269,48 @@ MFX_PACK_END()
 
 #endif
 
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+/* Multi-adapters Querying structs */
+typedef enum
+{
+    MFX_ENCODE_COMPONENT = 1,
+    MFX_DECODE_COMPONENT = 2,
+    MFX_VPP_COMPONENT    = 3
+} mfxComponentType;
```

Comment:
how FEI is classified?

Reply:
FEI is ENCODE + extension buffer (attached to ```mfxVideoParam``` on Init). On HEVC currently it is separate plugin, but it is possible to initialize it same way (with ext buffer) as AVC. We just need to add such buffer to API for HEVC.

Reply:
But for HEVC FEI scope of supported configurations is absolutely same as for regular HEVC encoder, so it is not critical issue.